ui/app/styles/components/icon.scss

```diff
@@ -3,6 +3,35 @@
  * SPDX-License-Identifier: MPL-2.0
  */
 
+.icon {
+  align-items: center;
+  display: inline-flex;
+  height: 1.5rem;
+  justify-content: center;
+  width: 1.5rem;
+}
+
+.icon-blue {
+  color: $blue;
+}
+
+.file-icon {
+  align-items: center;
+  display: flex;
+  height: 1em;
+  justify-content: center;
+  margin-right: 0.5em;
```

Comment:
An overall note on sizing, when we have the sizing var I use it for new css. I am not going through this file and replacing the sizes that can be replaced by variables, however. I will do that in a later PR because there are large gaps in our spacing vars. e.g. we use em or rems in a lot of components but no matching vars exist for them.